### ISSUE TITLE
The addon stopped working

### ISSUE THREAD

**Issue by sorin-pop** [anxdpanic/plugin.video.youtube#248]
Yesterday I noticed that nothing seems to work in the Youtube addon. Whenever I go to some menu point (e.g. Subscriptions) it just can't go further and show the following options, it hangs. So practically it's impossible to get anywhere within the addon. I am on a Raspberry Pi with Libreelec.

For now I would just like to know: anybody else experiencing this by any chance?

Kind regards.

**Comment by Psych85** [comment]
Yes I have the same experience. RPI4 4gb with Libreelec 10.0.1. Hope the wonderful, and much smarter people here, are able to find a resolution. As until they do my kids are on my back Hahahah.

**Comment by sorin-pop** [comment]
OK, anybody else having this issue?.... (I guess if it would be widespread, more people would write? could it be an only-Raspberry Pi issue?)

**Comment by BehnK** [comment]
I'm on a Pi 2 running LibreElec v9 (Kodi Leia) and it's fine and dandy (although I'm running YouTube 6.8.18 alpha2 due to the buffering issue from a few weeks back). My Win10 laptop is running the stock version of YouTube (6.8.18 matrix.1) and it's also working fine. Sounds likely that it's OS related.

**Comment by shankarinece** [comment]
This issue is seen in Kodi, running on OSMC. Youtube plugin v6.8.18+matrix.1, Kodi v19.1 running on RPi3 and on RPi4.

**Comment by sorin-pop** [comment]
I'm running LibreElec 9.2.8. Maybe I should try YouTube 6.8.18 alpha2 (like @BehnK )?

**Comment by shankarinece** [comment]
Is anyone else facing this issue? I tried with different older versions of youtube plugin on recent versions of LE and OSMC. I could not able to make it work. The search takes longer time than usual - approximately ~4 to 5 minutes.

**Comment by sorin-pop** [comment]
I am still facing this issue. Wherever I go, whatever menu point I want to step into (e.g. Subscriptions), I just see the spinning icon. I never waited for 4-5 minutes though... I'll try that. Anyway this certainly is very inconvenient (unusable) for me like this. I also uninstalled the addon and reinstalled it. Didn't help.

Also, since very few people seem to react, it seems this is not such a widespread issue...

**Comment by sorin-pop** [comment]
I was able to remedy the situation by doing a complete reset of my LibreElec + Kodi. Now the Youtube addon works fine again (but I had to reinstall all my other addons as well,and reconfigure again everything). So it was probably just something wrong in my Kodi system (not sure what), and most probably not a general issue (Which was confirmed by the small number of posts on this ticket - it really didn't look like many people are suddenly affected by the same issue).

Closing.
Thank you.
